airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/test/commands.py

```diff
@@ -30,27 +31,27 @@
     type=bool,
     is_flag=True,
 )
-@click.option(
-    "--fast-tests-only",
-    help="When enabled, slow tests are skipped.",
-    default=False,
-    type=bool,
-    is_flag=True,
-)
 @click.option(
     "--concurrent-cat",
     help="When enabled, the CAT tests will run concurrently. Be careful about rate limits",
     default=False,
     type=bool,
     is_flag=True,
 )
+@click.option(
+    '--skip-step',
+    '-x',
+    multiple=True,
+    type=str,
+    help="Skip a step by name. Can be used multiple times to skip multiple steps.",
+)
 @click.pass_context
 async def test(
```

Comment:
Shall we add a `discover` subcommand or `--dry-run` flag to help users discover the available step names?

Reply:
Once I make this change you suggested
https://github.com/airbytehq/airbyte/pull/32188#discussion_r1384937430

Then --help should allow them to discover the step names